### PR TITLE
See app logs on CG

### PR DESCRIPTION
**Pull request by davemcorwin**
## Changes proposed in this pull request:
- Adds actual app logging to the proxy

This currently doesn't add any information not already surfaced from the CF router logs, BUT now it works and we can add more detail as we see fit.

## security considerations
None

### CODE REVIEW

File: nginx.conf [hunk 1]
```diff
@@ -5,7 +5,10 @@ events { worker_connections 1024; }
 
 http {
   charset utf-8;
-  log_format cloudfoundry '$http_x_forwarded_for - $http_referer - [$time_local] "$request" $status $body_bytes_sent';
```
Comment (svenaas) [inline nginx.conf:8]: Are you deliberately eliminating the http_x_forwarded_for header? I've found that one useful for troubleshooting in past situations, but maybe between CF and CDN it isn't?

**Review by svenaas — APPROVED** [review]
One question posted, but this looks good to me.

File: nginx.conf (hunk 1 above)
Comment (davemcorwin) [reply nginx.conf:8]: It's already included in the router logs, as is some of the info we're still including in the app logs.

Comment (svenaas) [reply nginx.conf:8]: That'll do then. Thanks!

**Pull request merged**